doc/cookbook/using-free-client/UsingFreeClient.lhs

```diff
@@ -0,0 +1,173 @@
+# Using Free Client (for tests)
```

Comment:
"Inspecting and debugging clients (`Servant.Client.Free`)" would I think be a lot clearer and appeal a lot more to people who are looking to do just that =) Thoughts?